### DISCUSSION TITLE
Feel Free To Ask Questions Here

### DISCUSSION THREAD

**Discussion by sbtoonz** [sbtoonz/Trader_2.0#8] [Q&A]
Have a question about YML formatting?
Maybe a suggestion?

Cool feel free to start a chat here :D

**Comment by Mysternal** [comment]
hi, is there a way to silence knarr? I have him in my house and I don't want him to talk every time I go near him

**Reply by sbtoonz** [reply]
https://github.com/sbtoonz/Trader_2.0/commit/f2537d9d81666e67fef2a12c4bbf71c4369f1970


added configuration option for audio cue's

**Reply by sbtoonz** [reply]
https://valheim.thunderstore.io/package/OdinPlus/KnarrTheTrader/

**Reply by Mysternal** [reply]
Thank u so much!
